plugin/Renderer/InputFieldDeleteRecaptchaButtonRenderer.php

```diff
@@ -0,0 +1,39 @@
+<?php
```

Comment:
I'm not sure if this renderer is necessary.
What do you think to put this button inside of InputFieldPassworkRenderer and maybe rename it? Or do you think this is bad practice and every single element has to be a seperated class?